app/_locales/en/messages.json

```diff
@@ -792,6 +792,12 @@
   "confirmRecoveryPhrase": {
     "message": "Confirm Secret Recovery Phrase"
   },
+  "confirmTitleDescSignature": {
+    "message": "Only sign this message if you fully understand the content and trust the requesting site."
```

Comment:
I believe single line sentences should not contain a period. cc @coreyjanssen https://www.notion.so/MetaMask-Content-Design-Style-Guide-7d0b7579786540afbe02780ebb203254?pvs=4#7ffcee3343bc4115991c7a7e0d2f1c71

![Screenshot 2024-01-09 at 4 41 22 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/9c270f17-3355-4350-ab1c-346845d7488c)

Reply:
lgtm! updated d6f448e191195ee96d29050f45c7c8e9272d55c4

thanks @georgewrmarshall